lib/bootstrap.ts

```diff
@@ -1,9 +1,9 @@
+require("./common/bootstrap"); // note: here the order is important!
+$injector.require("staticConfig", "./config");
+$injector.require("config", "./config");
 require("./options");
```

Comment:
I think you should move userSettingsService before requiring options. When we require options we resolve errors, which resolves analyticsService, which is trying to resolve userSettingsService
```Typescript
$injector.require("config", "./config");
$injector.require("userSettingsService", "./services/user-settings-service");
require("./options");
```
In case you do not do this, you'll receive an error:
Error while reporting exception: Error: unable to resolve userSettingsService (I've executed $ appbuilder emulate android --gosho)